sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/HashAggregateExec.scala

```diff
@@ -313,7 +316,7 @@ case class HashAggregateExec(
       TaskContext.get().taskMemoryManager(),
       1024 * 16, // initial capacity
       TaskContext.get().taskMemoryManager().pageSizeBytes,
-      false // disable tracking of performance metrics
+      true // tracking of performance metrics
```

Comment:
Always turn it on? 

If we decide to always turn it on, why we still keep this parm?